roles/jenkins_job_launcher/tasks/main.yml

```
@@ -0,0 +1,39 @@
+---
+# tasks file for jenkins-job-launcher
+    
+- name: Trigger jenkins job
+  ansible.builtin.uri:
+    url: "{{ jjl_job_url }}/buildWithParameters/"
+    method: POST
+    user: "{{ jjl_username }}"
+    password: "{{ jjl_token }}"
+    force_basic_auth: true
+    status_code: [200,201]
```

This should get you :heavy_check_mark: in the linting check
```suggestion
    status_code: [200, 201]
```